L1Trigger/L1TMuon/python/simGmtStage2Digis_cfi.py

```diff
@@ -12,7 +12,7 @@
     forwardTFInput = cms.InputTag("simEmtfDigis", "EMTF"),
     #triggerTowerInput = cms.InputTag("simGmtCaloSumDigis", "TriggerTower2x2s"),
     triggerTowerInput = cms.InputTag("simGmtCaloSumDigis", "TriggerTowerSums"),
-    autoBxRange = cms.bool(False), # if True the output BX range is calculated from the inputs and 'bxMin' and 'bxMax' are ignored
+    autoBxRange = cms.bool(True), # if True the output BX range is calculated from the inputs and 'bxMin' and 'bxMax' are ignored
```

Comment:
@thomreis It looks like that due to this change we now lost lots of re-emulated muons for abs(eta) > 1.2
See https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideL1TOfflineDev#Development_Integration
and comparison plots for v24.2 vs v24.1
(This PR integrated is tagged as l1t-integration-v24.2)